From 2f93c1cb147a8e6da54d81efbfd06440fdfca828 Mon Sep 17 00:00:00 2001
Message-Id: <2f93c1cb147a8e6da54d81efbfd06440fdfca828.1413837857.git.chang-joon.lee@intel.com>
In-Reply-To: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
References: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
From: Deepak S <deepak.s@intel.com>
Date: Thu, 21 Aug 2014 12:01:17 +0530
Subject: [PATCH 246/312] FROM_UPSTREAM [VPG]: drm/i915: Enable gt powersave
 for cherryview

Issue: GMIN-3325
Change-Id: I781f27bfa4b4deea623ea25bd9559db5b44a9239
Signed-off-by: Deepak S <deepak.s@intel.com>
Acked-by: Ben Widawsky <ben@bwidawsk.net>
Reviewed-by: Mika Kuoppala <mika.kuoppala@intel.com>
Signed-off-by: Daniel Vetter <daniel.vetter@ffwll.ch>
Upstream-repo: git://anongit.freedesktop.org/drm-intel
Upstream-commit: 38807746fa2ce44b79957ff07813d10fcaf3d311
---
 drivers/gpu/drm/i915/intel_pm.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_pm.c b/drivers/gpu/drm/i915/intel_pm.c
index d8ceb3f..e30f995 100644
--- a/drivers/gpu/drm/i915/intel_pm.c
+++ b/drivers/gpu/drm/i915/intel_pm.c
@@ -5661,7 +5661,7 @@ void intel_enable_gt_powersave(struct drm_device *dev)
 		ironlake_enable_rc6(dev);
 		intel_init_emon(dev);
 		mutex_unlock(&dev->struct_mutex);
-	} else if (IS_GEN6(dev) || IS_GEN7(dev) || IS_BROADWELL(dev)) {
+	} else if (INTEL_INFO(dev)->gen >= 6) {
 		/* Do not enable turbo in BDW C (or older) steppings */
 		if (IS_BROADWELL(dev)) {
 			pci_read_config_dword(dev->pdev, GEN8_SRID_0_2_0_PCI, &val);
-- 
1.7.9.5

